common/lib/xmodule/xmodule/course_module.py

```diff
@@ -830,6 +830,13 @@ class CourseFields(object):
         },
         scope=Scope.settings
     )
+    allow_unsupported_xblocks = Boolean(
+        display_name=_("Allow the creation of unsupported XBlocks"),
```

Comment:
@sstack22 and @lamagnifica How about "Allow the Creation of Unsupported Problems and Tools"?

The thing is that this setting only impacts whether or not you can create new ones. It doesn't impact whether existing unsupported components render/work. I'm afraid that without "Creation", it reads that you are toggling whether or not unsupported components work in the course.